include/z64scene.h

```diff
@@ -534,119 +534,119 @@ typedef union {
 } SceneCmd; // size = 0x8
 
 typedef enum {
-    /* 0x00 */ SCENE_20SICHITAI2,
+    /* 0x00 */ SCENE_20SICHITAI2, // Southern Swamp (Clear)
     /* 0x01 */ SCENE_UNSET_1,
     /* 0x02 */ SCENE_UNSET_2,
     /* 0x03 */ SCENE_UNSET_3,
     /* 0x04 */ SCENE_UNSET_4,
     /* 0x05 */ SCENE_UNSET_5,
     /* 0x06 */ SCENE_UNSET_6,
-    /* 0x07 */ SCENE_KAKUSIANA,
-    /* 0x08 */ SCENE_SPOT00,
+    /* 0x07 */ SCENE_KAKUSIANA, // Lone Peak Shrine & Grottos
+    /* 0x08 */ SCENE_SPOT00, // Cutscene Scene
     /* 0x09 */ SCENE_UNSET_9,
-    /* 0x0A */ SCENE_WITCH_SHOP,
-    /* 0x0B */ SCENE_LAST_BS,
-    /* 0x0C */ SCENE_HAKASHITA,
-    /* 0x0D */ SCENE_AYASHIISHOP,
+    /* 0x0A */ SCENE_WITCH_SHOP, // Magic Hags' Potion Shop
+    /* 0x0B */ SCENE_LAST_BS, // Majora's Lair
+    /* 0x0C */ SCENE_HAKASHITA, // Beneath the Graveyard
+    /* 0x0D */ SCENE_AYASHIISHOP, // Curiosity Shop
     /* 0x0E */ SCENE_UNSET_E,
     /* 0x0F */ SCENE_UNSET_F,
-    /* 0x10 */ SCENE_OMOYA,
-    /* 0x11 */ SCENE_BOWLING,
-    /* 0x12 */ SCENE_SONCHONOIE,
-    /* 0x13 */ SCENE_IKANA,
-    /* 0x14 */ SCENE_KAIZOKU,
-    /* 0x15 */ SCENE_MILK_BAR,
-    /* 0x16 */ SCENE_INISIE_N,
-    /* 0x17 */ SCENE_TAKARAYA,
-    /* 0x18 */ SCENE_INISIE_R,
-    /* 0x19 */ SCENE_OKUJOU,
-    /* 0x1A */ SCENE_OPENINGDAN,
-    /* 0x1B */ SCENE_MITURIN,
-    /* 0x1C */ SCENE_13HUBUKINOMITI,
-    /* 0x1D */ SCENE_CASTLE,
-    /* 0x1E */ SCENE_DEKUTES,
-    /* 0x1F */ SCENE_MITURIN_BS,
-    /* 0x20 */ SCENE_SYATEKI_MIZU,
-    /* 0x21 */ SCENE_HAKUGIN,
-    /* 0x22 */ SCENE_ROMANYMAE,
-    /* 0x23 */ SCENE_PIRATE,
-    /* 0x24 */ SCENE_SYATEKI_MORI,
-    /* 0x25 */ SCENE_SINKAI,
-    /* 0x26 */ SCENE_YOUSEI_IZUMI,
-    /* 0x27 */ SCENE_KINSTA1,
-    /* 0x28 */ SCENE_KINDAN2,
-    /* 0x29 */ SCENE_TENMON_DAI,
-    /* 0x2A */ SCENE_LAST_DEKU,
-    /* 0x2B */ SCENE_22DEKUCITY,
-    /* 0x2C */ SCENE_KAJIYA,
-    /* 0x2D */ SCENE_00KEIKOKU,
-    /* 0x2E */ SCENE_POSTHOUSE,
-    /* 0x2F */ SCENE_LABO,
-    /* 0x30 */ SCENE_DANPEI2TEST,
+    /* 0x10 */ SCENE_OMOYA, // Mama's House & Barn
```

Comment:
Mama's house? where did that name come from? Would farm house or homestead make more sense in case someone confused it with mayor's residence (Letter to Mama is common slang for the Priority mail)

Reply:
Straight from the game
![image](https://user-images.githubusercontent.com/12245827/188237391-5be99c07-bbcd-446f-b04a-0b4b3f2910c8.png)

It was changed to "Ranch House" in PAL, but that change did NOT carry over to the US GameCube version. Screenshot courtesy of TCRF (based on research done by me 12 years ago lol):
![image](https://user-images.githubusercontent.com/12245827/188237552-6e10bf7c-a451-4e6d-aced-f6b7e770b0a4.png)

In the US version of MM3D, it's *still* "Mama's House":
![image](https://user-images.githubusercontent.com/12245827/188237834-c557ab7c-1b94-4598-9166-26533f170d38.png)

We are decompiling the US version of the game, and prefer to use US English over British English (based on some British-isms I've seen changed in some PRs). Thus, I think this should stay exactly as it is. It's the name in all US versions of the game.

Reply:
I'd say include both, it's not like we don't plan to do PAL as well. Interestingly this might be a mistranslation anyway, the Japanese is written with the kanji from https://en.wikipedia.org/wiki/Omoya, but with a space between them.